addons/l10n_in/models/template_in.py

```diff
@@ -9,10 +9,10 @@ class AccountChartTemplate(models.AbstractModel):
     @template('in')
     def _get_in_template_data(self):
         return {
-            'property_account_receivable_id': 'p10040',
-            'property_account_payable_id': 'p11211',
-            'property_account_expense_categ_id': 'p2107',
-            'property_account_income_categ_id': 'p20011',
+            'property_account_receivable_id': 'account_26010000',
+            'property_account_payable_id': 'account_16010000',
+            'property_account_expense_categ_id': 'account_54010000',
+            'property_account_income_categ_id': 'account_51010000',
             'code_digits': '6',
```

Comment:
```suggestion
            'code_digits': '8',
```
This will expand all account codes to 8 digits.